.prettierignore

```diff
@@ -0,0 +1,8 @@
+# Ignore artifacts:
+.netlify
+_data
```

Comment:
I think also `public` (Gatsby output folder) should get added to this list too?

Reply:
Exactly, I'll add it right away